src/Magento/FunctionalTestingFramework/Console/GenerateTestsCommand.php

```diff
@@ -152,7 +161,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $output->writeln("Generate Tests Command Run");
+        if (!empty($errMessages)) {
+            foreach (array_unique($errMessages) as $errMessage) {
+                $output->writeln($errMessage);
+                $output->writeln("\nGenerate Tests Command Run");
```

Comment:
to indicate all tests didn't generate, can the error message be tweaked:
`Generate Tests Command Run (partial)` or
`Generate Tests Command Run (with failures)`

Reply:
Liked the 2nd version. Fixed